### PR TITLE
Runes: document parameter usage, and let them be ints.

### PR DESCRIPTION
**Pull request by rustyrussell**
(based on #6294 )

Closes: #6288 

I had forgotten that we already strip punctuation from parameter names, so we merely need to treat integer parameters correctly FTW.  And document that `_` stripping!

### CODE REVIEW

File: plugins/commando.c [hunk 1]
```diff
@@ -427,6 +427,18 @@ static const char *check_condition(const tal_t *ctx,
 	if (!ptok)
 		return rune_alt_single_missing(ctx, alt);
 
+	/* Pass through valid integers as integers. */
+	if (ptok->type == JSMN_PRIMITIVE) {
```
Comment (niftynei) [inline plugins/commando.c:431]: cool this is much nicer than my suggestion./

**Review by niftynei — APPROVED** [review]
ACK 0fa24123ef833155396afb8935448ca87bf8ecb1

**Review by vincenzopalazzo — APPROVED** [review]
ACK https://github.com/ElementsProject/lightning/pull/6295/commits/0fa24123ef833155396afb8935448ca87bf8ecb1

**Review by vincenzopalazzo — APPROVED** [review]
ACK https://github.com/ElementsProject/lightning/pull/6295/commits/0fa24123ef833155396afb8935448ca87bf8ecb1 

Nice!

**Comment by rustyrussell** [comment]
Marked command stresstest as slow, to prevent timeouts.

Ack https://github.com/ElementsProject/lightning/pull/6295/commits/b1e2786b61785b872b54d39a7665ef4dd18c1d57

**Review by vincenzopalazzo — APPROVED** [review]
ACK https://github.com/ElementsProject/lightning/pull/6295/commits/b29732e43776303951cb94f29e0998c6867f6cf9

**Pull request merged**